ui/src/parseError.ts

```diff
@@ -78,11 +89,12 @@ function parseIfJson(o: any): any {
 }
 
 function getMessage(o: any, defaultMessage: string): string {
-    return (o && (o.message || o.Message)) || defaultMessage;
+    return (o && (o.message || o.Message || (typeof parseIfJson(o.body) === 'string' && o.body))) || defaultMessage;
```

Comment:
Number('') = 0
Number(null) = 0
etc.

I think you want parseInt

Reply:
Shouldn't matter for those two cases since we already check `if (errorType) {` the line above, but changed anyways since I don't have a strong pref.

Reply:
There are probably other cases, the point was it tries to convert into a number so is probably overpermissive.